checks/evaluation/signed_releases.go

```diff
@@ -47,11 +47,22 @@ func SignedReleases(name string,
 		f := &findings[i]
 
 		// Debug release name
-		releaseName, err := getReleaseName(f)
-		if err != nil {
-			e := sce.WithMessage(sce.ErrScorecardInternal, "could not get release name")
-			return checker.CreateRuntimeErrorResult(name, e)
+		if f.Outcome == finding.OutcomeNotApplicable {
+			dl.Warn(&checker.LogMessage{
+				Text: "no releases found",
+			})
```

Comment:
optional: this is redundant with the reason string, and could probably be removed

Reply:
👍